lib/charts/chart.rb

```diff
@@ -17,20 +17,34 @@ def validate_arguments(data, options)
     raise ArgumentError.new('Data missing') if data.empty?
```

Comment:
Hey Victor,
it would help me a lot if you can take a look at this Argument Validation Method! I was puzzling a lot how to write the ArgumentError statements more descriptive and to find better names for respective methods.